ui/src/containers/Search/Search.test.tsx

```diff
@@ -1,8 +1,48 @@
+import { TextInput } from '@patternfly/react-core';
+import { assert } from '../../store/utils';
+import { mount } from 'enzyme';
+import { when } from 'mobx';
 import React from 'react';
-import renderer from 'react-test-renderer';
+import { act } from 'react-dom/test-utils';
 import Search from '.';
+import { FakeHub } from '../../api/testutil';
+import { createProviderAndStore } from '../../store/root';
 
-it('should render the search component', () => {
-  const tree = renderer.create(<Search />).toJSON();
-  expect(tree).toMatchSnapshot();
+const TESTDATA_DIR = `src/store/testdata`;
+const api = new FakeHub(TESTDATA_DIR);
+const { Provider, root } = createProviderAndStore(api);
+
+describe('search resources', () => {
+  it('should render the search component', () => {
+    const component = mount(
```

Comment:
can we use `shallow` over here since `mount` is used for integration tests and here we are checking only the snapshot ?